src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -291,9 +291,9 @@ trait MarathonConf
   //Internal settings, that are not intended for external use
   lazy val internalStoreBackend = opt[String](
     "internal_store_backend",
-    descr = "The backend storage system to use. One of zk, mesos_zk, mem.",
+    descr = "The backend storage system to use. One of zk, mesos_zk, mem, new_zk, new_mem.",
```

Comment:
terms like new and old get old pretty quick.
Do we still have the idea to make the "what is zk" today available?
Perhaps we should rename like:
zk->twitter_zk
mem-> (remove)
new_zk -> zk
new_mem -> mem